src/main/java/com/newrelic/jfr/toevent/JITCompilationMapper.java

```diff
@@ -27,13 +28,7 @@
     var timestamp = event.getStartTime().toEpochMilli();
     var duration = event.getDuration();
     var attr = new Attributes();
-    var method = (RecordedMethod) event.getValue("method");
-    if (method != null) {
-      var methodDesc = method.getType().getName() + "." + method.getName() + method.getDescriptor();
-      attr.put("desc", methodDesc);
-    } else {
-      attr.put("desc", "[missing]");
-    }
+    attr.put("desc", StackTraceBlob.describeMethod((RecordedMethod) event.getValue("method")));
```

Comment:
This usage introduces a strange/unexpected dependency from JITCompilationMapper into StackTraceBlob.  Maybe we could move `describeMethod` into a new thing like `RecordedMethodSupport` or something?

Reply:
Fixed.